hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/graphite/WebTargetGraphiteClient.java

```diff
@@ -25,8 +25,8 @@
     public WebTargetGraphiteClient(WebTarget webTarget) {
         this.webTarget = webTarget
                 .path("render")
-                .queryParam("from", "-1minutes")
-                .queryParam("until", "now")
+                .queryParam("from", "-5minutes")
```

Comment:
That's what I mean - can we tell if graphite returned 0 because it's zero, or because it's filling for missing data? If we can't then maybe take last non-zero value or output zero if missing.